compiler/AST/primitive.cpp

```diff
@@ -96,18 +101,12 @@ returnInfoDefaultInt(CallExpr* call) {
   return returnInfoInt64(call);
 }
 
-static Type*
-returnInfoUInt64(CallExpr* call) {
-  return dtUInt[INT_SIZE_64];
-}
-
```

Comment:
This one makes a bit more sense to me at its new location when looking at the rest of the definitions.